lib/b2b/passwords_email.ts

```diff
@@ -0,0 +1,217 @@
+   * magic links are enabled for the member. If
+   *       you have not set a default `login_redirect_url` and magic links are not enabled for the member, an
+   * error is returned.
+   */
+  login_redirect_url?: string;
+  /**
+   * Used to determine which language to use when sending the user this delivery method. Parameter is a
+   * [IETF BCP 47 language tag](https://www.w3.org/International/articles/language-tags/), e.g. `"en"`.
+   *
+   * Currently supported languages are English (`"en"`), Spanish (`"es"`), and Brazilian Portuguese
+   * (`"pt-br"`); if no value is provided, the copy defaults to English.
+   *
+   * Request support for additional languages
+   * [here](https://docs.google.com/forms/d/e/1FAIpQLScZSpAu_m2AmLXRT3F3kap-s_mcV6UTBitYn6CdyWP0-o7YjQ/viewform?usp=sf_link")!
+   *
+   */
+  locale?: "en" | "es" | "pt-br";
+  /**
+   * Use a custom template for reset password emails. By default, it will use your default email template.
+   * The template must be a template using our built-in customizations or a custom HTML email for Magic Links
+   * - Reset Password.
+   */
+  reset_password_template_id?: string;
+}
+
+// Response type for `Email.resetStart`.
+export interface B2BEmailResetStartResponse {
+  /**
+   * Globally unique UUID that is returned with every API call. This value is important to log for debugging
+   * purposes; we may ask for this value to help identify a specific API call when helping you debug an issue.
+   */
+  request_id: string;
+  // Globally unique UUID that identifies a specific Member.
+  member_id: string;
+  // Globally unique UUID that identifies a member's email
+  member_email_id: string;
+  /**
+   * The HTTP status code of the response. Stytch follows standard HTTP response status code patterns, e.g.
+   * 2XX values equate to success, 3XX values are redirects, 4XX are client errors, and 5XX are server errors.
+   */
+  status_code: number;
+}
+
+export class Email {
+  private fetchConfig: fetchConfig;
+
+  constructor(fetchConfig: fetchConfig) {
+    this.fetchConfig = fetchConfig;
+  }
+
+  /**
+   * Initiates a password reset for the email address provided. This will trigger an email to be sent to the
+   * address, containing a magic link that will allow them to set a new password and authenticate.
+   *
+   * This endpoint adapts to your Project's password strength configuration.
+   * If you're using [zxcvbn](https://stytch.com/docs/passwords#strength-requirements), the default, your
+   * passwords are considered valid
+   * if the strength score is >= 3. If you're using
+   * [LUDS](https://stytch.com/docs/passwords#strength-requirements), your passwords are
+   * considered valid if they meet the requirements that you've set with Stytch.
+   * You may update your password strength configuration in the
+   * [stytch dashboard](https://stytch.com/dashboard/password-strength-config).
+   */
+  resetStart(
```

Comment:
There are a few JSDoc annotations that might be nice to add 
- `@param` - links to a given param
- `@returns` - links to a given param
- `@async` - declares the function returns a promise
- `@throws` - links to an error the function can throw

Some of these are redundant from the type information, but the type information requires a typescript compiler / lang server to be required, and not every dev environment may have that. TSDoc is a bit older and a bit better supported.

Adding these for this endpoint would look like

```js
   * @param data {@link B2BEmailResetStartRequest}
   * @async
   * @returns {@link B2BEmailResetStartResponse}
   * @throws A {@link StytchError} on a non-2xx response from the Stytch API
   * @throws A {@link RequestError} when the Stytch API cannot be reached
```

and gives me the following output on Webstorm - 
<img width="291" alt="image" src="https://github.com/stytchauth/stytch-node/assets/89937743/d294d09f-c573-4002-b0b4-8560f525d338">


would be nice to check with VSCode too!